app/views/categories/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class='form-head'>
+    <%= link_to categories_path do %>
+      <i class="fa fa-arrow-left"></i>
+    <% end %>
+    <h2>Categories</h2>
+    <div class="actions">
+      <%= button_to "Sign Out", destroy_user_session_path, method: :delete, class:"btn",style: 'color:white; background-color:transparent;' %>
+    </div>
+  </div>
+<% if @categories.nil? %>
+      <h3>Add a Category</h3>
+    <% else %>
+  <div id="list-con">
+   <% @categories.each do |category| %>
+      <%= link_to category_payments_path(category.id) do %>
+         <%= render category %>
+    <% end %>
+   <% end %>
+  </div>
+<% end %>
+
+<%= link_to "Add category", new_category_path, class: 'btn w-100 p-2', style: 'color:white; background-color:#5fb523;' %>
```

Comment:
- [ ] Please move the "Add Category" button to the bottom of the screen to match the given design.
- [ ] Please add a white outline around the button to match the original button design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/Kccendana/budget_App/assets/62290545/97ee08f7-ae78-43d8-9803-91ed4ffb2e99) | ![image](https://github.com/Kccendana/budget_App/assets/62290545/c36f0799-7be4-4890-b7c1-dddc36077372) |